examples/spark2/src/main/java/org/apache/carbondata/examples/sdk/CarbonReaderExample.java

```diff
@@ -33,127 +29,129 @@
 import org.apache.carbondata.sdk.file.Field;
 import org.apache.carbondata.sdk.file.Schema;
 
+import org.apache.commons.io.FileUtils;
```

Comment:
this line need to place in front of packege 'org.apache.carbondata', right?

Reply:
No, org.apache.carbondata is in front of this.